src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,77 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useMemo } from 'react';
+import { breadcrumbStyles, breadcrumbLinkStyles } from './Breadcrumb.style';
+import useTheme from 'hooks/useTheme';
+import BreadcrumbItem from './BreadcrumbItem/BreadcrumbItem';
+import isEmpty from 'lodash/isEmpty';
+import uniqueId from 'lodash/uniqueId';
+import { Link } from 'react-router-dom';
+import BreadcrumbCollapsed from './BreadcrumbCollapsed/BreadcrumbCollapsed';
+import { getLastDataItemProperty } from '../../utils/helpers';
+import { BreadcrumbItemData } from './types';
+
+export type Props = {
+  /** Defines the data for constructing the related breadcrumb items */
+  data: BreadcrumbItemData[];
+};
+
+const Breadcrumb: React.FC<Props> = props => {
+  const { children, data = [] } = props;
+  const theme = useTheme();
+  const passDataToRouterLink = ({ to, label }: BreadcrumbItemData) => (
```

Comment:
you can move this outside the breadcrumb as you will need it only once

Reply:
if I move this outside of the breadcrumb, I will have to convert this util function to a functional component. Therefore, I will have to create another function to use it for the data mapping. 

example: 
```
data.map( ( {label, to}: BreadcrumbItemData) => <PassDataToRouteLink label={label} to={to} /> );
```
We could try wrapping the passDataToRouterLink with a useCallback hook but i am still not sure if we gain performance value.